Initial Scene for Implementation of the QuestManager

saiedbaran opened the pull request.

saiedbaran committed:
Generate Folder Structure

saiedbaran committed:
Create Scene for QM implementation

michaelholly commented on ProjectSettings/ProjectVersion.txt:
```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.4.16f1
+m_EditorVersion: 2018.4.19f1
```
please use 2018.4.16

michaelholly committed:
QMController to generate Quest Manager

saiedbaran committed:
Add Node Graph Editor for Quest generation

Now we can create main quests and sub quests using Node graph, It is easier to create and modify. Also for the scripts we can now drag and drop script files into graphs.

Also, Logo of Gear group is added to our script

michaelholly committed:
CMController Setup PlaceHolder
Unity 2018.4.20 Update

michaelholly committed:
added nodegraph reader
clean up

michaelholly committed:
create SubQuest objects from NodeGraph
StringValue attribute for Enums

michaelholly committed:
auto position quests
quest text

michaelholly committed:
refactored QMReader

michaelholly committed:
generic method for instantiate quest objects

saiedbaran committed:
Change FBX scale to be unit scale

michaelholly committed:
Merge branch 'develop' into Development/QuestManager

michaelholly committed:
Merge remote-tracking branch 'remotes/origin/develop' into Development/QuestManager

Petz27 committed:
QuestManager Update
- QuestData holds information about a quest
- QuestView shows quests to user (PC & VR)

Petz27 committed:
QM refactor:
- remove dead code (Starboard) and unnecessary 3rd party stuff
- changed MaroonCoin position in UI
- changed quest position after completing it

Petz27 committed:
Last bugfixes + project cleanup

Petz27 committed:
Project cleanup

Petz27 committed:
Project cleanup - variable names
